pvlib/modelchain.py

```diff
@@ -623,7 +623,7 @@ def effective_irradiance_model(self):
             fd*self.total_irrad['poa_diffuse'])
         return self
 
-    def complete_irradiance(self, times=None, weather=None):
+    def complete_irradiance(self, times=None, weather=None, **kwargs):
```

Comment:
Done. 
I had to set a default value for the clearsky_tolerance and chose 1.1, again according to an early comment from @adriesse. Would you suggest to have the clearsky_tolerance as an input into the complete_irradiance function, so it can be changed by the user or should the user write his/her own function in case he/she wants to change those parameters?